src/content/ja/2020/capabilities.md

```diff
@@ -0,0 +1,328 @@
+
+```js
+const relatedApps = await navigator.getInstalledRelatedApps();
+relatedApps.forEach((app) => {
+  console.log(app.id, app.platform, app.url);
+});
+```
+
+{{ figure_markup(
+  image="get_installed_related_apps.png",
+  caption="getInstalledRelatedApps()を使用しているページ数",
+  description="HTTPArchiveで監視しているページ数に基づくgetInstalledRelatedApps()の使用状況のグラフです。モバイルデバイスとデスクトップデバイスでの使用状況を比較しています。モバイルデバイスでは着実な成長を示しており、2020年10月にはデスクトップの44ページに対して363ページとピークを迎えています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1774881171&format=interactive",
+  sheets_gid="860146688",
+  sql_file="get_installed_related_apps_usage.sql"
+  )
+}}
+
+2020年の間に、`getInstalledRelatedApps()`APIはモバイルWebサイトで着実な成長を示しています。10月には、HTTP Archiveによって追跡された363のモバイルページがこのAPIを利用していました。デスクトップページでは、APIの成長はそれほど速くありません。これは現在AndroidストアとMicrosoftストアがWindows向けに提供しているアプリよりもかなり多くのアプリを提供していることにも起因している可能性があります。
+
+## コンテンツインデックスAPI
+
+Webアプリは、キャッシュストレージやインデックスDBなど、さまざまな方法でオフラインでコンテンツを保存できます。しかし、ユーザーにとっては、どのコンテンツがオフラインで利用可能かを発見するのは難しい。<a lang="en" href="https://web.dev/content-indexing-api/">コンテンツインデックスAPI</a>（<a lang="en" href="https://wicg.github.io/content-index/spec/">WICGエディター草案</a>）を利用することで、開発者はコンテンツをより目立つように露出させることができます。現在このAPIをサポートしているブラウザはAndroidのChromeだけです。このブラウザでは、ダウンロードメニューに「あなたのための記事」の一覧が表示されます。コンテンツインデックスAPIでインデックスされたコンテンツがそこに表示されます。
+
+コンテンツインデックスAPIは、新しい`ContentIndex`インターフェイスを提供することでサービスワーカーAPIを拡張します。このインターフェイスは、サービスワーカーの登録の`index`プロパティで利用できます。`add()`メソッドを使うと、開発者はコンテンツをインデックスに追加できます。各コンテンツには、ID、URL、起動URL、タイトル、説明、アイコンのセットが必要です。オプションで、コンテンツを記事、ホームページ、動画などの異なるカテゴリにグループ化できます。`delete()` メソッドはインデックスからコンテンツを再び削除し、`getAll()` メソッドはインデックス化されたすべてのエントリのリストを返す。
+
+{{ figure_markup(
+  image="content_indexing_api.png",
+  alt="コンテンツインデックスAPIを使用したChromeでのページロードの割合",
+  caption='コンテンツインデックスAPIを使用したChromeでのページロードの割合<br>（出典 <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">コンテンツインデックス</a>)',
+  description="Chromeでこの機能を使用しているページロードの割合に基づく、コンテンツインデックスAPIの使用状況のグラフです。最初は比較的低い使用率を示していますが、2020年10月になると突然10倍に増え、Chromeでのページロードの0.0021%で使用されています",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=258329620&format=interactive",
+  sheets_gid="626752011"
+  )
+}}
+
+コンテンツインデックスAPIは、2020年7月にChrome 84で提供が開始されました。提供直後は、Chromeでのページロードの約0.0002%の間にAPIが使用されていました。2020年10月時点では、この値はほぼ10倍に増加しています。
+
+## 新しいトランスポートAPI
+
+最後に、現在オリジントライアル中の2つの新しいトランスポート方式があります。1つ目は開発者がWebSocketsで高頻度のメッセージを受信できるようにするもので、2つ目はHTTPやWebSocketsとは別に全く新しい双方向通信プロトコルを導入しています。
+
+### WebSocketsのBackpressure
```

Comment:
Titles need special handling so the span tags are not included in the generated link:

```suggestion
### <span lang="en">WebSockets</span>の<span lang="en">Backpressure</span> {websocketsのbackpressure}
```

Reply:
Thanks, I fixed.